components/infobox/extensions/commons/placement_stats_settings.lua

```diff
@@ -0,0 +1,12 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:InfoboxPlacementStats/Settings
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	tiers = { '1', '2', '3', },
+	exclusionTypes = { 'Qualifier', },
+}
```

Comment:
config files should be as human readable as possible, even for non programmers. Could you add detailed comments to both of these vars, to explain what these do?